### DISCUSSION TITLE
Dynamic Trace Sample Rate

### DISCUSSION THREAD

**Discussion by codeforaliving** [open-telemetry/opentelemetry-java-instrumentation#6789] [Q&A]
Hello,

We currently have a custom extension with a SpanProcessor.  For traces, we are using the following config:

`otel.traces.sampler.arg=1.0`

Is there a way to reload the sampler arg at runtime?  We went down the path of implementing a custom `TraceIdRatioBasedSampler` but we are not sure how to get the updated sample rate from our Spring Boot app into a dependency and ultimately to the extension JAR.  We are using version 1.14.0. Thanks in advance.

**Comment by mateuszrzeszutek** [comment]
Linking the answer from another thread: https://github.com/open-telemetry/opentelemetry-java/discussions/4504#discussioncomment-3794571
